pandas/tests/groupby/test_rank.py

```diff
@@ -710,3 +710,9 @@ def test_rank_categorical():
 
     expected = df.astype(object).groupby("col1").rank()
     tm.assert_frame_equal(res, expected)
+
+def test_groupy_op_with_nullables():
```

Comment:
Minor: typo in method name: `test_groupy_op_with_nullables`-> `test_groupby_op_with_nullables`